forms/common-themes/src/main/resources/theme/base/admin/resources/partials/required-actions.html

```diff
@@ -20,8 +20,8 @@
         <tbody>
         <tr ng-repeat="requiredAction in requiredActions" data-ng-show="requiredActions.length > 0">
             <td>{{requiredAction.name}}</td>
-            <td><input type="checkbox" ng-model="requiredAction.enabled" ng-change="updateRequiredAction(requiredAction)"></td>
-            <td><input type="checkbox" ng-model="requiredAction.defaultAction" ng-change="updateRequiredAction(requiredAction)"></td>
+            <td><input type="checkbox" ng-model="requiredAction.enabled" ng-change="updateRequiredAction(requiredAction)" id="{{requiredAction.alias}}"></td>
```

Comment:
A better approach here would be to put the id="{{requiredAction.alias}}" on the tr. Then add name="enabled" and name="defaultAction" to the corresponding input fields. That would make it easier to create XPath expressions that are relatively simple and reliable.